src/mixins/audio_token_behavior.mixin.js

```diff
@@ -65,5 +65,74 @@
         this.onPlay();
       }
     },
+
+    // for most objects, all 'controls' are the same size, for audio_tokens we need to treat them uniquely
+    _setCornerCoords: function() {
+      var coords = this.oCoords,
+          newTheta = fabric.util.degreesToRadians(45 - this.angle),
+          cosTheta = fabric.util.cos(newTheta),
+          sinTheta = fabric.util.sin(newTheta),
+          controlSize = this.cornerSize;
+
+      for (var control in coords) {
+        switch (control) {
+          case 'deleteControl':
+            controlSize = this.deleteControlSize;
+            break;
+          case 'playControl':
+            controlSize = this.playControlSize;
+            break;
+          default:
+            // 'corner' is fabrics default term for controls...because they are in the corners?
+            controlSize = this.cornerSize;
+        }
```

Comment:
The code in the base Object_behavior_mixin looks nearly identical except for this switch in the beginning of the loop.